Card/src/main/java/com/braintreepayments/api/card/BaseCard.kt

```diff
@@ -0,0 +1,116 @@
+package com.braintreepayments.api.card
+
+import androidx.annotation.RestrictTo
+import com.braintreepayments.api.core.PaymentMethod
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Base class used to build various types of cards
+ */
+abstract class BaseCard(
```

Comment:
This is out of scope for this PR, but we should make a note to remove the `BaseCard` class. We removed `UnionPayCard`, so we should only need a single `Card` type in `v5`.

Reply:
Thought the same way, PaymentMethod will also be removed.